pkg/nsx/services/securitypolicy/firewall.go

```diff
@@ -2,12 +2,12 @@ package securitypolicy
 
 import (
 	"sync"
-
+	
```

Comment:
Please check your patch preview before requesting review.

Reply:
Sorry, I didn't know how it happened. Updated.